### ISSUE TITLE
new armor for dragons

### ISSUE THREAD

**Issue by WilliamBlaze** [AlexModGuy/Ice-and-Fire-Suggestions#56]
as the title implies, i request we add more armor types for our dragons. we can craft silver for ourselves but currently we can not make a silver armor for our dragons? 

and what of leather dragon armor; so that we may dye the armor to our choice of colors to decorate our dragons as we see fit?

i could go one step further and say we include the ability to make armor types for our dragons using existing ores in our game provided from other mods; meaning copper armors,  cobalt armors, this list can go on' but i think i made my suggestion clear.

**Comment by Wolf7104** [comment]
I like the idea, but "requests" aren't well accepted by developers. 
And you can simply re-texture the current Dragon Armor and suggest that. It's not that hard, most people will make something good in a few tries.

**Comment by WilliamBlaze** [comment]
poorly worded to call it a request on my part i know, but it appears that alex might have already been on top of this idea or maybe i've just gotten the wrong impression from viewing this https://github.com/Alex-the-666/Ice_and_Fire/issues/21

**Comment by LordDecay26** [comment]
yeah, i already suggested this in his discord. he must have found that interesting enough to see it to be worked on.
